python/lsst/afw/cameraGeom/utils.py

```diff
@@ -315,28 +386,42 @@ def getAmpImage(self, amp, imageFactory):
 
 
 class ButlerImage(FakeImageDataSource):
-    """A class to return an Image of a given Ccd using the butler"""
+    """A class to return an Image of a given Ccd using the butler.
+
+    Parameters
+    ----------
+    butler : `lsst.daf.persistence.Butler` or None
+        The butler to use. If None, an empty image is returned.
+    type : `str`
+        The type of image to read (e.g. raw, bias, flat, calexp).
+    isTrimmed : `bool`
+        If true, the showCamera command expects to be given trimmed images.
+    verbose : `bool`
+        Be chatty (in particular, print any error messages from the butler).
+    background : `float`
+        The value of any pixels that lie outside the CCDs.
+    gravity : `str` or None
+        If the image returned by the butler is trimmed (e.g. some of the
+        SuprimeCam CCDs), specify how to fit the image into the available
+        space; N => align top, W => align left.
```

Comment:
`"N"`, `"W"`. Are those the only options?

Reply:
I have no idea.

Reply:
We need to find a middle ground here.

If we're undertaking a basically mechanical reformatting of documentation, then I don't think it's wise to let it get bogged down by trying to fix all the defects in the documentation which is being reformatted.

However, it does seem reasonable to use this opportunity to review the documentation for problems and ticket them to be addressed later.

Reply:
DM-13417 was filed to make `cameraGeom.utils.makeImageFromCamera`'s documentation more useable. It turned into numpydoc-ifying it since I was modifying the docs anyway. I did not intend to dig into every parameter or fix docstring docstrings that required more than a cursory look at the code.

Certainly for our future "move to numpydoc" work, if we require that the doxygen->numpydoc step *also* fix all badly written or misleading docs (many of which never underwent code review in the first place!), it'll never get finished.

In this instance, it doesn't appear that the `gravity` parameter is used anywhere! So, I guess I'll delete it and its docstring?